stable/magento/requirements.yaml

```diff
@@ -2,3 +2,4 @@ dependencies:
 - name: mariadb
   version: 0.7.0
```

Comment:
lets upgrade to the latest mariadb chart version as well. wdyt?

Reply:
done